pet-routing/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './App.css';
+import { Route, Switch } from "react-router-dom";
+import OneDogImage from "./component/RandomDogImage"
+import NavBar from './component/NavBar'; 
+import MultipleDogImages from "./component/MultipleDogImages";
+import RandomBreed from "./component/RandomBreed";
+function App() {
+  return (
+    <div className="App">
+    <NavBar/>
+    <Switch>
+      <Route exact path={"/dog/random"} component={OneDogImage}/>
+      <Route path={"/dog/random/:num"} component={MultipleDogImages}/>
+      <Route path={"/dog/:breed"} component={RandomBreed}/>
+      <Route component={Error} />
```

Comment:
Need to create and import an Error component to render if there is an error, you would also have the route path to `*`